styles/global.css

```diff
@@ -62,4 +62,8 @@
 }
 .prose code::before {
   content: "";
+}
+
+.team-container img {
+  margin-bottom: 0;
 }
```

Comment:
You should be able to do this with `mb-0`, no? In general with tailwind, you almost never have to add new CSS.

Reply:
Indeed! I thought I tried it and it didn't work, it might have been an issue with the hot reload? I changed it now.